infrastructure/lambda/template.yaml

```diff
@@ -106,6 +110,10 @@ Globals:
         - '{{resolve:secretsmanager:${SecretArn}:SecretString:JAVA_LAYER}}' # or NODEJS_LAYER or PYTHON_LAYER
         - SecretArn: !FindInMap [ EnvironmentConfiguration, !Ref Environment, dynatraceSecretArn ]
     AutoPublishAlias: live
+    DeploymentPreference:
+      Enabled: true
+      Type: !Ref LambdaDeploymentPreference
```

Comment:
Are we leaving the default as AllAtOnce for all envs, or choosing a specific strategies? (probably one to discuss with the teams/coordinate across the CRIs)